beacon/versioned_beacon_state.go

```diff
@@ -0,0 +1,19 @@
+package beacon
+
+import (
+	"errors"
+
+	"github.com/attestantio/go-eth2-client/spec"
+	"github.com/attestantio/go-eth2-client/spec/capella"
+)
+
+func HistoricalSummaries(state *spec.VersionedBeaconState) ([]*capella.HistoricalSummary, error) {
```

Comment:
Created a new file for this, idea is that there may be other containers we want to access that attestant will not implement on their spec.VersionedBeaconState implementation

Reply:
do they not have a getter for this like .Slot() and .Validators()?

Reply:
Nope :(